### PR TITLE
Yizha/load uncorrelated events

### PR DESCRIPTION
**Pull request by yingzhao-1**

### CODE REVIEW

File: Domain/Sia.Domain/Event.cs [hunk 1]
```diff
@@ -4,6 +4,7 @@
 using System.Collections.Generic;
 using System.Dynamic;
 using System.Text;
+using System.Linq;
```
Comment (macrobug) [inline Domain/Sia.Domain/Event.cs]: this `using` doesn't seem to be related to the change below.

Comment (yingzhao-1) [reply Domain/Sia.Domain/Event.cs]: you are right, this was for previous version of code. Removing it.

File: Domain/Sia.Domain/Event.cs [hunk 2]
```diff
@@ -15,5 +11,6 @@ public class Event : IEntity, IJsonDataObject
         public DateTime Occurred { get; set; }
         public DateTime EventFired { get; set; }
         public object Data { get; set; }
+        public string PrimaryTicketId { get; set; }
     }
-}
+}
```
Comment (pdimitratos) [inline Domain/Sia.Domain/Event.cs]: Git needs a newline at the end of each file.
